### ISSUE TITLE
Map Color Auto Update Cause Crash with Resource Packs

### ISSUE THREAD

**Issue by aRedPenguin** [daveyliam/mapwriter#37]
When using a resource pack with Map Colours set to Auto, MapWriter crashes. Setting the Map Colours to Frozen when not using a resource pack allows for play. Once it is set to frozen you can change back to your texture pack.

Here is the crash log: http://pastebin.com/stmkfSfJ

**Comment by Vectron** [comment]
what resource pack do you use?
and what modpack are you using?

**Comment by darktrek** [comment]
im having the same issue as aredpenguin.

modpack im using is [the 1.7.10 modpack on the technic launcher] 
resource packs [Sphax PureBDcraft 32x32] http://bdcraft.net/purebdcraft-minecraft
                         [The 1.7.10 Modpack patch] http://bdcraft.net/community/pbdc-patches-rel-packs/technic-the-modpack-t3729.html

**Comment by aRedPenguin** [comment]
Yeah, I'm using Faithful. It seems that when mapwriter/opis is updated, the configs reset and the issue resumes.

**Comment by Vectron** [comment]
i'm unable to reproduce this bug ,tried with twothe's version of mapwriter, opis and my own. all seem to work perfectly with the mentioned modpack and resourcepack.
it might be something odd on your end but cant realy say what.

would it be posible to post the full log and not the crash report?

**Comment by darktrek** [comment]
i have loaded using the mapwriter 1.7.10 2.1.2 and still gets the crash

crash report http://pastebin.com/48azEY2X

complete log part 1 http://pastebin.com/uN54xssV
                     part 2 http://pastebin.com/2vHFH9qc
                     part 3 http://pastebin.com/Mjjb7GQz

had to do the log in 3 parts due to its size and paste-bins limits

**Comment by Vectron** [comment]
hmm, what i think what is happening:
mapwriter starts normaly creating block textures from (alot) blocks. the games skippes ticks becouse it's lagging to much and thus fires a chunck load event. mapwriter then tries to create a image for the new chunck but fails becoes the texture creating isnt done yet.

now for a fix i dont have 1 without updating mapwriter (i will look into this but cant promise anything)

**Comment by Vectron** [comment]
fixed in this version:

https://www.dropbox.com/s/s4mzbyfuozc6pbu/MapWriter-1.7.10-2.1.5-Beta-2.jar?dl=0
